### ISSUE TITLE
The bot IS the issue.

### ISSUE THREAD

**Issue by ehannum** [alekxeyuk/Skribbl.io-Bot#20]
Who wants to bet this ass hole is just destroying skribble.io because he can't draw and seeing people have fun drawing makes him mad?

**Comment by ybogasyad** [comment]
"My try at creating fully automated game bot.

Features
Asynchronous, event based code style
Line optimization for faster drawing
Supports two different dither algos
Easy image size control
Automatic googling, and image picking
Search filters (image size, colors, etc...)
Chat bot
And many more ..."
from alekxeyuk himself. people like you are the issue. People who dont understand that there is a difference between legit cheating, ragehacking, and fun cheating. People like you dont understand that some of us find the mechanisms behind a game interesting. Its educting to deconstruct a game and find loopholes that you werent meant to find. Hence why he says this is for educational purposes only. You can tell that he just chose this game at random. He couldve done any other game but still would get the generic backlash of morons saying "cheating cuz bad". youre so typical, and its sad that youll never understand the findamentals of legit cheating, (cheating to win aka being a dick,) or fun cheating (cheating to show off what youre capable of and the games weaknesses.) hope youll change some day though

**Comment by YellowJustice** [comment]
> "My try at creating fully automated game bot.
> 
> Features
> Asynchronous, event based code style
> Line optimization for faster drawing
> Supports two different dither algos
> Easy image size control
> Automatic googling, and image picking
> Search filters (image size, colors, etc...)
> Chat bot
> And many more ..."
> from alekxeyuk himself. people like you are the issue. People who dont understand that there is a difference between legit cheating, ragehacking, and fun cheating. People like you dont understand that some of us find the mechanisms behind a game interesting. Its educting to deconstruct a game and find loopholes that you werent meant to find. Hence why he says this is for educational purposes only. You can tell that he just chose this game at random. He couldve done any other game but still would get the generic backlash of morons saying "cheating cuz bad". youre so typical, and its sad that youll never understand the findamentals of legit cheating, (cheating to win aka being a dick,) or fun cheating (cheating to show off what youre capable of and the games weaknesses.) hope youll change some day though

stfu lib

**Comment by ybogasyad** [comment]
> > "My try at creating fully automated game bot.
> > Features
> > Asynchronous, event based code style
> > Line optimization for faster drawing
> > Supports two different dither algos
> > Easy image size control
> > Automatic googling, and image picking
> > Search filters (image size, colors, etc...)
> > Chat bot
> > And many more ..."
> > from alekxeyuk himself. people like you are the issue. People who dont understand that there is a difference between legit cheating, ragehacking, and fun cheating. People like you dont understand that some of us find the mechanisms behind a game interesting. Its educting to deconstruct a game and find loopholes that you werent meant to find. Hence why he says this is for educational purposes only. You can tell that he just chose this game at random. He couldve done any other game but still would get the generic backlash of morons saying "cheating cuz bad". youre so typical, and its sad that youll never understand the findamentals of legit cheating, (cheating to win aka being a dick,) or fun cheating (cheating to show off what youre capable of and the games weaknesses.) hope youll change some day though
> 
> stfu lib

nice meme

**Comment by GrimReaperFloof** [comment]
> > > "My try at creating fully automated game bot.
> > > Features
> > > Asynchronous, event based code style
> > > Line optimization for faster drawing
> > > Supports two different dither algos
> > > Easy image size control
> > > Automatic googling, and image picking
> > > Search filters (image size, colors, etc...)
> > > Chat bot
> > > And many more ..."
> > > from alekxeyuk himself. people like you are the issue. People who dont understand that there is a difference between legit cheating, ragehacking, and fun cheating. People like you dont understand that some of us find the mechanisms behind a game interesting. Its educting to deconstruct a game and find loopholes that you werent meant to find. Hence why he says this is for educational purposes only. You can tell that he just chose this game at random. He couldve done any other game but still would get the generic backlash of morons saying "cheating cuz bad". youre so typical, and its sad that youll never understand the findamentals of legit cheating, (cheating to win aka being a dick,) or fun cheating (cheating to show off what youre capable of and the games weaknesses.) hope youll change some day though
> > 
> > 
> > stfu lib
> 
> nice meme

Lol. I like it when people do this interesting stuff like creating bots for a game, or what @ybogasyad calls "fun cheating". Fun cheating is cool. It interests me and gives me an idea of how the game works behind the scenes, and inspires me to see what I can do with it.

**Comment by Frotty** [comment]
Kids, no one has a problem if you write this bot and try it out for fun.
Having it always online and automatically join rooms, spam chat and be useless, is the actual problem.

**Comment by JackOfSpade** [comment]
> Kids, no one has a problem if you write this bot and try it out for fun.
> Having it always online and automatically join rooms, spam chat and be useless, is the actual problem.
> I.e. destroying the game for other players. It leaves the realm of "interesting" and goes into "douche".

I agree, the fact he keeps his computer on 24/7 to run multiple instances of this bot is beyond "testing".
